homeassistant/components/recorder/migration.py

```diff
@@ -363,6 +363,7 @@ def _apply_update(engine, new_version, old_version):
         if engine.dialect.name == "mysql":
             _modify_columns(engine, "events", ["event_data LONGTEXT"])
             _modify_columns(engine, "states", ["attributes LONGTEXT"])
+        _modify_columns(engine, "events", ["event_type VARCHAR(64)"])
```

Comment:
I'm pretty sure this won't work for sqlite

Reply:
Since copying all the data to a new table is likely going to be a very long running operation, I think we can exclude sqlite here.